### ISSUE TITLE
Error in console while using bow

### ISSUE THREAD

**Issue by AcesGamingUK** [JEFF-Media-GbR/Spigot-Drop2Inventory#24]
**Versions -** 
Paper: `1.16.1-R0.1-SNAPSHOT`
Drop2Inv: `3.9.2`
Drop2InvAPI: `1.13`

**Item -** 
Enchanted minecraft:bow `Power 3`
minecraft:arrow
```
09.07 08:10:25 [Server] ERROR Could not pass event EntityDeathEvent to Drop2Inventory v3.9.2
09.07 08:10:25 [Server] INFO java.lang.NullPointerException: null
09.07 08:10:25 [Server] INFO at de.jeff_media.Drop2Inventory.Utils.isMobEnabled(Utils.java:26) ~[?:?]
09.07 08:10:25 [Server] INFO at de.jeff_media.Drop2Inventory.Listener.onEntityDeath(Listener.java:59) ~[?:?]
09.07 08:10:25 [Server] INFO at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor65.execute(Unknown Source) ~[?:?]
09.07 08:10:25 [Server] INFO at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:793) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.EntityLiving.d(EntityLiving.java:1489) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.EntityLiving.die(EntityLiving.java:1417) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.EntityLiving.damageEntity(EntityLiving.java:1261) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.EntityMonster.damageEntity(EntityMonster.java:52) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.EntityArrow.a(EntityArrow.java:331) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.IProjectile.a(IProjectile.java:125) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.EntityArrow.tick(EntityArrow.java:175) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.EntityTippedArrow.tick(EntityTippedArrow.java:92) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.WorldServer.entityJoinedWorld(WorldServer.java:832) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.World.a(World.java:932) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.WorldServer.doTick(WorldServer.java:526) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1353) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1212) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1000) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:10:25 [Server] INFO at java.lang.Thread.run(Unknown Source) [?:1.8.0_162]
```

```
09.07 08:12:28 [Server] ERROR Could not pass event EntityDeathEvent to Drop2Inventory v3.9.2
09.07 08:12:28 [Server] INFO java.lang.NullPointerException: null
09.07 08:12:28 [Server] INFO at de.jeff_media.Drop2Inventory.Utils.isMobEnabled(Utils.java:26) ~[?:?]
09.07 08:12:28 [Server] INFO at de.jeff_media.Drop2Inventory.Listener.onEntityDeath(Listener.java:59) ~[?:?]
09.07 08:12:28 [Server] INFO at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor65.execute(Unknown Source) ~[?:?]
09.07 08:12:28 [Server] INFO at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:793) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.EntityLiving.d(EntityLiving.java:1489) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.EntityLiving.die(EntityLiving.java:1417) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.EntityLiving.damageEntity(EntityLiving.java:1261) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.EntityGhast.damageEntity(EntityGhast.java:48) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.EntityArrow.a(EntityArrow.java:331) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.IProjectile.a(IProjectile.java:125) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.EntityArrow.tick(EntityArrow.java:175) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.EntityTippedArrow.tick(EntityTippedArrow.java:92) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.WorldServer.entityJoinedWorld(WorldServer.java:832) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.World.a(World.java:932) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.WorldServer.doTick(WorldServer.java:526) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1353) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1212) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1000) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:12:28 [Server] INFO at java.lang.Thread.run(Unknown Source) [?:1.8.0_162]
```

```
09.07 08:13:15 [Server] ERROR Could not pass event EntityDeathEvent to Drop2Inventory v3.9.2
09.07 08:13:15 [Server] INFO java.lang.NullPointerException: null
09.07 08:13:15 [Server] INFO at de.jeff_media.Drop2Inventory.Utils.isMobEnabled(Utils.java:26) ~[?:?]
09.07 08:13:15 [Server] INFO at de.jeff_media.Drop2Inventory.Listener.onEntityDeath(Listener.java:59) ~[?:?]
09.07 08:13:15 [Server] INFO at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor65.execute(Unknown Source) ~[?:?]
09.07 08:13:15 [Server] INFO at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:793) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.EntityLiving.d(EntityLiving.java:1489) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.EntityLiving.die(EntityLiving.java:1417) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.EntityLiving.damageEntity(EntityLiving.java:1261) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.EntityMonster.damageEntity(EntityMonster.java:52) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.EntityArrow.a(EntityArrow.java:331) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.IProjectile.a(IProjectile.java:125) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.EntityArrow.tick(EntityArrow.java:175) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.EntityTippedArrow.tick(EntityTippedArrow.java:92) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.WorldServer.entityJoinedWorld(WorldServer.java:832) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.World.a(World.java:932) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.WorldServer.doTick(WorldServer.java:526) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1353) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1212) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1000) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:15 [Server] INFO at java.lang.Thread.run(Unknown Source) [?:1.8.0_162]
```
```
09.07 08:13:49 [Server] ERROR Could not pass event EntityDeathEvent to Drop2Inventory v3.9.2
09.07 08:13:49 [Server] INFO java.lang.NullPointerException: null
09.07 08:13:49 [Server] INFO at de.jeff_media.Drop2Inventory.Utils.isMobEnabled(Utils.java:26) ~[?:?]
09.07 08:13:49 [Server] INFO at de.jeff_media.Drop2Inventory.Listener.onEntityDeath(Listener.java:59) ~[?:?]
09.07 08:13:49 [Server] INFO at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor65.execute(Unknown Source) ~[?:?]
09.07 08:13:49 [Server] INFO at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at org.bukkit.craftbukkit.v1_16_R1.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:793) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.EntityLiving.d(EntityLiving.java:1489) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.EntityLiving.die(EntityLiving.java:1417) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.EntityLiving.damageEntity(EntityLiving.java:1261) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.EntityMonster.damageEntity(EntityMonster.java:52) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.EntityArrow.a(EntityArrow.java:331) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.IProjectile.a(IProjectile.java:125) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.EntityArrow.tick(EntityArrow.java:175) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.EntityTippedArrow.tick(EntityTippedArrow.java:92) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.WorldServer.entityJoinedWorld(WorldServer.java:832) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.World.a(World.java:932) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.WorldServer.doTick(WorldServer.java:526) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1353) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1212) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1000) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Paper-68]
09.07 08:13:49 [Server] INFO at java.lang.Thread.run(Unknown Source) [?:1.8.0_162]
```

**Comment by mfnalex** [comment]
Thank you, I will have a look at this now

**Comment by mfnalex** [comment]
Fixed in release 3.9.3
